public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -39,13 +40,31 @@ export function getFillColor(cfg: StyleConfigValues) {
   return undefined;
 }
 
+const textLabel = (cfg: StyleConfigValues) => {
+  const fontFamily = config.theme2.typography.fontFamily;
+  const textConfig = { 
```

Comment:
It seems like we are never using this value so TS is throwing an error (at least for me locally)

Reply:
👍  yeah I committed some suggestions directly on github without checking what else would need to be changed, but just made some changes locally